pkg/traceql/engine_metrics.go

```diff
@@ -17,9 +17,14 @@ import (
 	v1 "github.com/grafana/tempo/pkg/tempopb/common/v1"
 	"github.com/grafana/tempo/pkg/util"
 	"github.com/prometheus/prometheus/model/labels"
+	"github.com/willf/bloom"
 )
 
-const internalLabelBucket = "__bucket"
+const (
+	internalLabelBucket   = "__bucket"
+	maxExemplars          = 100
+	maxExemplarsPerBucket = 2
```

Comment:
Probably should be configurable. I preferred to go for a simple solution as we don't know what kind of sampling we're going to like best.